tests/testAutopas/tests/containers/verletListsCellBased/verletListsCells/PairwiseVerletListsTest.cpp

```diff
@@ -0,0 +1,173 @@
+/**
+ * @file PairwiseVerletListsTest.cpp
+ * @author tirgendetwas
+ * @date 22.11.20
+ */
+#include "PairwiseVerletListsTest.h"
+
+#include "autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h"
+using ::testing::_;
+using ::testing::Values;
+
+TEST_P(PairwiseVerletListsTest, testTwoParticles) {
```

Comment:
Please write a brief description of each test (what do you want to test, how do you test it, what do you expect,... 1-3 sentences).